templates/watcher/config/00-default.conf

```diff
@@ -0,0 +1,43 @@
+[DEFAULT]
+state_path = /opt/stack/data/watcher
+transport_url = {{ .TransportURL }}
+control_exchange = watcher
+debug = True
+
+[database]
+connection = {{ .DatabaseConnection }}
+
+[oslo_policy]
+policy_file = /etc/watcher/policy.yaml.sample
+
+[oslo_messaging_notifications]
+driver = messagingv2
+
+[keystone_authtoken]
+memcached_servers = {{ .MemcachedServers  }}
+# TODO jgilaber implement handling this option when we add tls support
+cafile = /opt/stack/data/ca-bundle.pem
```

Comment:
comment it out like the one on line 32 (plus chandan comment about /opt/stack)

Reply:
thanks, done!